EventListener/ExceptionLogSubscriber.php

```diff
@@ -12,6 +12,8 @@
 
 final class ExceptionLogSubscriber implements EventSubscriberInterface
 {
+    const DEFAULT_ERROR_MESSAGE = "No error message given";
```

Comment:
I think we should keep this constant private, as it is just a default message.

```suggestion
    private const DEFAULT_ERROR_MESSAGE = "No error message given";
```